domain/src/main/java/com/teamagam/gimelgimel/domain/base/logging/DomainLoggerFactoryHolder.java

```diff
@@ -0,0 +1,25 @@
+package com.teamagam.gimelgimel.domain.base.logging;
+
+public class DomainLoggerFactoryHolder {
+    private static DomainLoggerFactory sDomainLoggerFactory;
+
+    public static void initialize(DomainLoggerFactory domainLoggerFactory) {
+        if (isInitialized()) {
```

Comment:
It's static method, I don't like the exception here, espicially when the method "isInitialized" is private. Nobody can if it initialized, except the class itself.
You can initialize it when necessary and don't do it if someone did it already.